frontend/bot/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "coda-bot",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
```

Comment:
our license should be Apache 2 

Ref: @cmr

Reply:
thx, fixed (I'll make another PR to fix it in the other package.jsons)